.github/workflows/ci.yml

```diff
@@ -85,6 +85,17 @@ jobs:
           docs-dir: docs/build/html
           bucket-url: s3://cloudcustodian.io/docs
 
+  Docker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: c7n build and test
+        uses: ./.github/composites/docker-build-push
```

Comment:
this is missing the HUB_USER and HUB_TOKEN env vars, does this still push to dockerhub?